python/paddle/v2/fluid/tests/notest_csp.py

```diff
@@ -0,0 +1,40 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import paddle.v2.fluid as fluid
+
+
+class TestCSPFramework(unittest.TestCase):
```

Comment:
Could we make sure that this unit test doesn't break the CI before we finish the implementation of CSP?

Reply:
Sure, will do.

Reply:
@wangkuiyi I made sure that the CI doesn't break. Both TeamCity and Travis pass.